packaging/suse/release/release.sh

```diff
@@ -55,7 +55,7 @@ update_package() {
   echo "Setting version in _service file"
   cd $DEST_PROJECT/portus
   cp _service _service.orig
-  sed -e "s/master.tar.gz/$RELEASE.tar.gz/g" -i _service
+  sed -e "s/\/SUSE\/Portus\/archive\/.*.tar.gz/\/SUSE\/Portus\/archive\/$RELEASE.tar.gz/g" -i _service
```

Comment:
FYI: Instead of escaping the slashes, `sed` allows you to use a different separator, e.g. `|`, and therefore increase readability:
```
sed -e "s|/SUSE/Portus/archive/.*.tar.gz|/SUSE/Portus/archive/$RELEASE.tar.gz|g" -i _service
```